config/locales/en.yml

```diff
@@ -32,6 +32,11 @@ en:
         not_confirmed: Your email address could not be confirmed. Perhaps you have already completed this step?
         confirmation_sent: "Confirmation email sent!"
         confirmation_not_sent: "Could not send a confirmation email."
+  enterprise_mailer:
+    confirmation_instructions:
+      subject: "Please confirm the email address for %{enterprise}"
+    welcome:
+      subject: "%{enterprise} is now on "
```

Comment:
Can you include the sitename variable as well?

      subject: "%{enterprise} is now on %{sitename}"

And then pass it in the mailer:

    subject = t('enterprise_mailer.welcome.subject',
                enterprise: @enterprise.name,
                sitename: Spree::Config[:site_name])